scripts/components/RecentChanges/ChangeRow.js

```diff
@@ -22,30 +12,34 @@ export default function ChangeRow({
   action,
   afterEmoji,
 }) {
-  const classes = useStyles()
-  const theme = useTheme()
-  const matches = useMediaQuery(theme.breakpoints.up('sm'))
-
   return (
-    <TableRow>
-      <TableCell>
-        {eventIcon}
-        <span className={classes.span}>{`${titleCase(eventName)}d`}</span>
-      </TableCell>
-      <TableCell>
+    <Box display="flex" alignItems="center">
+      <Box display="flex" alignItems="center" minWidth="7.1rem">
+        <Box padding="0.5rem">
```

Comment:
Generic comment about file: See if you can move these to classes, custom components or variables so you don't declare the same things 3 times over or more, as that makes changing things later a pain